main.tex

```diff
@@ -107,7 +113,7 @@ \subsection{Formats, commands, and templates}
 \noindent
 \addtocounter{footnote}{1}%
 \footnotetext{%
-  The Markdown package uses Lua to parse markdown text. Whereas Lua\TeX{} can execute Lua code directly, other \TeX{} engines must use the shell of the operating system to execute Lua code. Since accessing the shell is a security risk, users must express their consent by writing \texttt{--shell-escape}.%
+  The Markdown package uses Lua to parse markdown text. Whereas Lua\TeX{} can execute Lua code directly, other \TeX{} engines must use the operating system shell to execute Lua code. Since accessing the shell is a security risk, users must express their consent by writing \texttt{-{}-shell-escape}.%
```

Comment:
I prefer "shell of the operating system" rather than "operating system shell". For the less technical reader, it separates the two concepts (shell and operating system), which makes the text easier to digest. The line breaking does not seem to be improved either way.

```suggestion
  The Markdown package uses Lua to parse markdown text. Whereas Lua\TeX{} can execute Lua code directly, other \TeX{} engines must use the shell of the operating system to execute Lua code. Since accessing the shell is a security risk, users must express their consent by writing \texttt{-{}-shell-escape}.%
```